content/blog/projects/finished/carmodel.md

```diff
@@ -0,0 +1,46 @@
+---
+lastmod: 2023-06-17T10:13:20Z
+head:
+  meta:
+    - name: keywords
+      content: projects, robots, finished
+    - name: robots
+      content: index follow
+    - name: author
+      content: Andrzej Piotrowski
+    - name: copyright
+      content: © 2023 KN Integra
+    - name: createdAt
+      content: 2023-03-20T19:40:00Z
+---
+
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/439fe2cf-ea08-42c0-b7b0-92a9ee60a5eb" alt="CarMODEL Logo" style="max-height: 300px;  display: block; margin: 0; padding-left:180px">

```

Comment:
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo